src/components/PinInput/src/PinInputControl.vue

```diff
@@ -209,16 +205,33 @@ export default {
 
 <style module="$s">
 .PinInputContainer {
+	position: relative;
+	width: var(--pin-input-width);
+}
+
+.PinInput {
+	position: absolute;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	z-index: 1;
```

Comment:
Does Maker have any z-indexing variables? Would be handy to ensure this isn't layered above/below other components

Reply:
Maker doesn't have z-indexing variables. They're not that useful anyway, as they only affect the relative z-ordering of items within a particular [stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context). Generally, where possible, we let the order of the items in the DOM dictate how things stack so that devs have control over this within their own apps, e.g.
```
<template>
  <!-- if an app is setup like this, dialogs will appear above modals -->
  <m-dialog-layer />
  <m-modal-layer />
  
  <!-- but if it's setup like this, modals will appear above dialogs -->
  <m-modal-layer />
  <m-dialog-layer />
</template>
```
and where it's not possible, or where we have a thing we know we want to just slightly be above other things even within a particular stacking context, we usually just slap a `z-index: 1` on it and call it a day, so I think it's fine here as well.

Reply:
Makes sense to me. Just wanted to see if there was an internal convention here. I agree that the layering concerns should be owned by the applications using Maker, generally speaking.

Reply:
Adjusted to use order of markup instead of explicit z-index